RELEASE.md

```diff
@@ -16,6 +16,10 @@ Further details on building, testing and use can be found in [README.md](https:/
 
 This is release candidate 1 of version 0.11.0 of oqs-provider which continues from the earlier 0.10.0 release. This release is fully tested to be used in conjunction with the main branch of [liboqs](https://github.com/open-quantum-safe/liboqs) and is guaranteed to be in sync with v0.15.0 of `liboqs`.
 
+
+### What's New
+Support for bp256_mlkem512, bp384_mlkem768, and bp512_mlkem1024.
```

Comment:
Would you be OK saying "Generic support for Brainpool-based hybrid PQ logic added"? Whether it's worth while mentioning that it's initially instantiated for MLKEM algs on the PQ side, I'd leave to you (or maybe @RodriM11 : worthwhile highlighting?).